segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -923,58 +923,58 @@ private Void handleException(long requestId, String segment, long offset, String
         };
 
         if (u instanceof StreamSegmentExistsException) {
-            log.info(requestId, "Segment '{}' already exists and cannot perform operation '{}'.",
-                     segment, operation);
+            log.info(requestId, "Segment '{}' already exists and cannot perform operation '{}'. Exception : {}",
```

Comment:
These will double-log the exception stack trace. No need to add an explicit `{}` to the message. Just put the exception as the last parameter.

Reply:
Also, some of these are well known exceptions; they can only come from one place and they are the result of user error. Adding a 20-40 line stack trace is going to significantly pollute the logs - that's the reason we removed them in the first place.